charts/kafka/chart/templates/kafka-operator.yaml

```
@@ -58,6 +68,16 @@ spec:
         metadata:
           labels:
             app: kafka-zookeeper
+        affinity:
+          podAntiAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              - labelSelector:
+                  matchExpressions:
+                    - key: app
+                      operator: In
+                      values:
+                        - kafka-zookeeper
```

The `matchLabels` syntax is a bit easier to read IMO:
```suggestion
                  matchLabels:
                    app: kafka-zookeeper
```